winui/Avatar-view/customization.md

```diff
@@ -146,7 +146,7 @@ avatarView.Background = new LinearGradientBrush()
 
 ## Font
 
-Customize the font of initials in the `AvatarView` control by using the `FontFamily` and `Foreground` properties.
```

Comment:
@MadhumithanJayapathi API references are wrong, take properties from here.

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control

Reply:
@Manivannan-E Removed the wrong API links and added API's from the link provided